server/plugin.go

```diff
@@ -367,6 +367,43 @@ func (p *Plugin) GetToDo(ctx context.Context, username string, githubClient *git
 	return text, nil
 }
 
+func (p *Plugin) HasUnreads(info *GitHubUserInfo) bool {
+	username := info.GitHubUsername
+	ctx := context.Background()
+	githubClient := p.githubConnect(*info.Token)
+	config := p.getConfiguration()
+
+	issues, _, err := githubClient.Search.Issues(ctx, getReviewSearchQuery(username, config.GitHubOrg), &github.SearchOptions{})
+	if err != nil {
+		mlog.Error(err.Error())
+		return false
+	}
+
+	notifications, _, err := githubClient.Activity.ListNotifications(ctx, &github.NotificationListOptions{})
+	if err != nil {
+		mlog.Error(err.Error())
+		return false
+	}
+
+	yourPrs, _, err := githubClient.Search.Issues(ctx, getYourPrsSearchQuery(username, config.GitHubOrg), &github.SearchOptions{})
+	if err != nil {
+		mlog.Error(err.Error())
+		return false
+	}
+
+	yourAssignments, _, err := githubClient.Search.Issues(ctx, getYourAssigneeSearchQuery(username, config.GitHubOrg), &github.SearchOptions{})
+	if err != nil {
+		mlog.Error(err.Error())
+		return false
+	}
+
+	if issues.GetTotal() == 0 && len(notifications) == 0 && yourPrs.GetTotal() == 0 && yourAssignments.GetTotal() == 0 {
```

Comment:
Would it make sense to only check the notifications? That way you only get the message if something has changed.
At least for me, this will never be false.

@jwilander Thoughts?

Reply:
Yes I was thinking that would be better originally but I was wondering if some people would still want the reminders even if there are no new unreads. e.g. if they read a PR but didn't review it because they didn't have time. That would make it less useful to me and you, though, and I think we are representative of the core users of the

@ivenk or @jasonblais any thoughts?

Reply:
I personally would appreciate reminders even if there are no new unreads. However i see why one might disagree.

Reply:
Ok. I am fine with leaving it as-is.